share_app/src/main/res/layout/activity_wifi.xml

```diff
@@ -10,7 +10,9 @@
     <RelativeLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_below="@id/toolbar">
+        android:layout_below="@id/toolbar"
```

Comment:
Please run `auto-format` or Ctrl + Shift + F (for eclipse key binding) to this file to auto-arrange the lines